### PR TITLE
Wrong definition of meteor.d.ts for Meteor.status()

### PR DESCRIPTION
**Pull request by stphdenis**
The actual definition is

``` js
declare module "meteor/meteor" {
  export module Meteor {
    function status(): Meteor.StatusEnum;
  }
}
```

instead of

``` js
declare module "meteor/meteor" {
  export module Meteor {
    interface Status {
      status: Meteor.StatusEnum;
      connected: boolean;
      retryCount: number;
    }
    function status(): Meteor.Status;
  }
}
```

### CODE REVIEW

File: meteor/meteor.d.ts [hunk 1]
```diff
@@ -1,435 +1,673 @@
 // Type definitions for Meteor 1.3
 // Project: http://www.meteor.com/
-// Definitions by: Dave Allen <https://github.com/fullflavedave>
+// Definitions by: Alex Borodach <https://github.com/barbatus>,
```
Comment (vvakame) [inline meteor/meteor.d.ts]: `// Definitions by: Dave Allen <https://github.com/fullflavedave>, Alex Borodach <https://github.com/barbatus>`
please use this format.

File: angular-meteor/angular-meteor.d.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@
 // Definitions by: Peter Grman <https://github.com/pgrm>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-/// <reference path="../meteor/meteor.d.ts" />
+/// <reference path="../meteor/meteor-1.2.d.ts" />
```
Comment (vvakame) [inline angular-meteor/angular-meteor.d.ts]: Why do you use version 1.2?

**Comment by mhegazy** [comment]
Thanks for your contribution. This PR has failing CI tests and can not be merged in at the time being. For housekeeping purposes we are closing stale PRs. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the `master` branch.

**Comment by fullflavedave** [comment]
I apologize for my silence on this.  As indicated in the recently revised README, these definitions are currently deprecated, and the best definitions for Meteor 1.3 can now be found here:  <https://github.com/meteor-typings/meteor>.  However, I plan to update the DefinitelyTyped Meteor definitions with those found in the meteor-typings/meteor repo.  The `Meteor.status()` definition will be updated in both repos.

**Pull request closed without merging**